.golangci.yml

```diff
@@ -0,0 +1,20 @@
+linters:
+  enable:
+    - nakedret 
+    - interfacer
+    - varcheck
+    - deadcode 
+    - structcheck 
+    - misspell
+    - maligned 
+    - ineffassign
+    - goconst
+    - goimports
+    - errcheck 
+    - dupl 
+    - unparam 
+    - golint 
+    - staticcheck
+    - unused
+    - gosimple
+  
```

Comment:
Also, see that it still with the simple. 
The files should have one 1 space at the end. 

@kensipe ^

Reply:
good catch!